checkov/terraform/checks/resource/aws/WAFRuleHasAnyActions.py

```diff
@@ -15,17 +15,26 @@ def scan_resource_conf(self, conf):
         rules = None
         if conf.get("rule") and isinstance(conf["rule"], list):
             rules = conf["rule"]
-        # dont blame me i didn't name one rule and the other rules and the other activated_rule
+        # don't blame me I didn't name one rule and the other rules and the other activated_rule
         elif conf.get("rules") and isinstance(conf["rules"], list):
             rules = conf["rules"]
         elif conf.get("activated_rule") and isinstance(conf["activated_rule"], list):
             rules = conf["activated_rule"]
 
         if isinstance(rules, list):
             for rule in rules:
+                passing = False
                 if "action" in rule and rule['action'] != [{}]:
-                    continue
+                    passing = True
                 if "override_action" in rule and rule['override_action'] != [{}]:
+                    passing = True
+
+                statements = rule.get('statement')
+                if statements and isinstance(statements, list):
+                    for statement in statements:
+                        if statement.get('managed_rule_group_statement'):
+                            passing = True
+                if passing:
```

Comment:
you can use here `if not passing: return CheckResult.FAILED` instead of `continue` and then return